orb/src/jobs/setup.yml

```diff
@@ -2,15 +2,11 @@ parameters:
   executor:
     default: default
     type: executor
-  npm-version:
-    default: '8'
-    type: string
```

Comment:
**issue**: deleting this parameter will be a breaking change as any project that was setting it previously will now error as the parameter doesn't exist anymore. this might technically be permissible as no project seems to be setting it currently, but it might be preferable to change the logic to default to `''` and then only run the command if the parameter isn't the empty string. so it will use the bundled npm version by default whilst still allowing users to override it.

Reply:
I have revieved the value and added a deprecation description.